app/lib/efile/nc/d400_schedule_s_calculator.rb

```diff
@@ -0,0 +1,28 @@
+module Efile
+  module Nc
+    class D400ScheduleSCalculator < ::Efile::TaxCalculator
+      attr_reader :lines, :value_access_tracker
+
+      def initialize(value_access_tracker:, lines:, intake:)
+        @value_access_tracker = value_access_tracker
+        @lines = lines
+        @intake = intake
+      end
+
+      def calculate
+        set_line(:NCD400_S_LINE_27, :calculate_line_27)
+        set_line(:NCD400_S_LINE_41, :calculate_line_41)
+      end
+
+      private
+
+      def calculate_line_27
+        @intake.tribal_wages_amount.to_i
```

Comment:
For my understanding -- it's not possible for this value to be blank at this point b/c that's not permitted in the db, correct? so here is where we're just rounding to an integer

Reply:
the value can be blank at this point

Reply:
if so, do we need the use `&.to_i`?

Reply:
yeah that makes sense!

Reply:
i don't think so because when called on nil, `to_i `returns 0

Reply:
oh nice